cpp/src/barretenberg/stdlib/primitives/bigfield/bigfield.hpp

```diff
@@ -312,6 +312,20 @@ template <typename Composer, typename T> class bigfield {
         prime_basis_limb.fix_witness();
     }
 
+    /**
+     * @brief Get the binary basis limbs of a bigfield scalar.
+     *
+     * @return vector of the binary basis limbs (circuit type)
+     */
+    std::vector<field_t<Composer>> get_limbs() const
```

Comment:
If you intend to use these limbs in some way, you need to forcefully reduce the element. Otherwise there could be several limb values representing the same underlying value.